contracts/exchange/ibetEscrow.sol

```diff
@@ -0,0 +1,422 @@
+/**
```

Comment:
[IMO]If you follow convention, fileName might be UpperCamel.

Reply:
oops